_src/static/sass/main-ltr.scss

```diff
@@ -13,17 +13,21 @@
 @import '../../vendor/bi-app/bi-app-ltr';   // LTR support
 
 // ------------------------------
-// #UTILITIES
```

Comment:
I moved BASE above UTILITIES because we're including `_variables.scss` in our BASE set, which should be compiled before any utilities that reference them. Is this incorrect?